edb/tools/test/__init__.py

```diff
@@ -88,9 +88,11 @@
               help='Maintain a running time log file at FILEPATH')
 @click.option('--list', 'list_tests', is_flag=True,
               help='list all the tests and exit')
+@click.option('--postgres-dsn', type=str,
+              help='Test ')
```

Comment:
```suggestion
              help='Use the specified Postgres cluster instead of starting a temporary local one.')
```

Reply:
Oops, shouldn't have multi-tasked. Thanks for the fix!